sdk/core/azure-core-experimental/src/main/java/com/azure/core/experimental/util/BinaryData.java

```diff
@@ -47,15 +48,20 @@
  */
 public final class  BinaryData {
     private static final ClientLogger LOGGER = new ClientLogger(BinaryData.class);
+
+    private static JsonSerializer defaultJsonSerializer;
```

Comment:
The property name may not be complying with coding styles.

Reply:
it does not complain.